admin-dev/themes/new-theme/js/pages/product/services/paginated-combinations-service.ts

```diff
@@ -111,6 +111,6 @@ export default class PaginatedCombinationsService implements PaginationServiceTy
   }
 
   private getFilterId(): string {
-    return `product_combinations_${this.productId}_${this.shopId}`;
```

Comment:
shop is no longer needed ? 🤔

Reply:
No, adding it in the first place wasn't needed, the idea was to rely on a filter ID depending on shop ID so that it is associated with shop context. But the filter's DB storage already handles an id_shop column so it was overkill